pdb2sql/StructureSimilarity.py

```diff
@@ -252,34 +238,27 @@ def compute_irmsd_fast(self,izone=None,method='svd',cutoff=10,check=True):
         # read the izone file
         if izone is None:
             resData = self.compute_izone(cutoff,save_file=False)
-        elif not os.path.isfile(izone):
-            resData = self.compute_izone(cutoff,save_file=True,filename=izone)
+        # elif not os.path.isfile(izone):
```

Comment:
why comment that ?

Reply:
`izone` is used as both input file and output file, which is not unique. I left a TODO comment around that to add output file to the function.